launcher/solOSX.sh

```diff
@@ -1,2 +1,14 @@
 #!/usr/bin/env bash
-lwjreOSX/bin/java -XstartOnFirstThread -jar libs/solDesktop.jar -noSplash
+
+JRE=lwjreOSX/bin/java
+ARCH=$(uname -m)
+if [[ "$ARCH" == "x86_64" ]]; then
+  JRE=lwjreOSX/bin/java
+elif [[ "$ARCH" == "arm64" ]]; then
+  JRE=lwjreOSArm/bin/java
```

Comment:
There's a typo here, the directory should be `lwjreOSXArm`

Reply:
I've corrected it. Thanks for letting me know.